src/index.html

```diff
@@ -123,9 +123,15 @@ <h2>&rarr; Twitter</h2>
 
       <h1>Code of Conduct</h1>
 
-      <p><strong>All</strong> of our events follow a code of conduct that you can read at: <a href="https://github.com/jsoxford/summerofhacks.github.io/blob/master/CODEOFCONDUCT.md">github.com/jsoxford/summerofhacks.github.io/blob/master/CODEOFCONDUCT.md</a></p>
-
-      <p>It's our absolute priority that attendees are safe and are feel respected. If you have any issues, concerns or questions then don't hesitate to contact us.</p>
+      <p><strong>All</strong> of our events follow a code of conduct that you can read at
+        <a href="https://github.com/summer-of-hacks/soh19website/blob/master/CODEOFCONDUCT.md">
+          https://github.com/summer-of-hacks/soh19website/blob/master/CODEOFCONDUCT.md</a></p>
```

Comment:
Changed the link so that it references the COC in our own GH account.

> It's the same as the JSOxford one but this gives us that brand dis-association from JSOxford.